chalice/utils.py

```diff
@@ -78,6 +78,10 @@ def create_zip_file(source_dir, outfile):
 class OSUtils(object):
     ZIP_DEFLATED = zipfile.ZIP_DEFLATED
 
+    def environ(self):
+        # type: () -> Dict[str,str]
+        return os.environ.copy()
```

Comment:
Should we be caching the copy of the os.environ on the first call and then return the cached copy on subsequent calls? Mainly just wondering in the case that if we update the environement variables that gets returned, the update will not actually get returned again if `OSUtils.environ` gets called again.

Reply:
Not needed for this particular use case but that is a reasonable expectation of someone consuming the `OSUtils` class in general so I will add it.